src/components/accounts/CreateAccount.js

```diff
@@ -83,27 +94,39 @@ class CreateAccount extends Component {
         const { accountId, token } = this.state;
         const { match, createNewAccount, setFormLoader } = this.props
 
-        const fundingKey = match.params.fundingKey;
         const fundingContract = match.params.fundingContract;
-
-        this.setState({ loader: true });
+        const fundingKey = match.params.fundingKey;
+        setLinkdropData({ fundingContract, fundingKey })
         
-        try {
-            await createNewAccount(accountId, fundingKey, fundingContract, token)
-        } finally {
-            this.setState({ loader: false });
-            setFormLoader(false)
-        }
+        setTempAccount(accountId)
+        setFormLoader(false)
         
-        this.handleCreateAccountSuccess();
-    }
-
-    handleCreateAccountSuccess = () => {
-        const { accountId } = this.state;
-
-        this.props.refreshAccount();
+        this.setState({ loader: true });
         let nextUrl = process.env.DISABLE_PHONE_RECOVERY === 'yes' ? `/setup-seed-phrase/${accountId}` : `/set-recovery/${accountId}`;
         this.props.history.push(nextUrl);
+
+        /********************************
+        Deprecated create account here in favor of creating account after recovery is confirmed
```

Comment:
Let's not keep commented out code in repository.

Reply:
Agree. We changed a lot about this flow and I think we were just being careful with rebase.

Reply:
Removed by: https://github.com/near/near-wallet/pull/700/commits/dbed5937f56d58f0c64fc61f90fb9580f7c81909